engines/bladerunner/script/ai/free_slot_a.cpp

```diff
@@ -544,7 +544,7 @@ void AIScriptFreeSlotA::checkIfOnBridge() {
 	float x, y, z;
 	Actor_Query_XYZ(kActorFreeSlotA, &x, &y, &z);
 	// bug? this should probably check if McCoy is close enough because bridge will break long after rat died and player tries to walk through
-	if (-150.0 <= x && x < -30.0f) {
+	if (-150.0 <= x && x < -30.0f && !BLADERUNNER_INDESTRUCTIBLE_BRIDGE) {
```

Comment:
replace with "easy" settings  _vm->_settings->getDifficulty() == kGameDifficultyEasy and do it only when BLADERUNNER_ORIGINAL_BUGS  is set to 0